internal/pkg/data/sql-datastore-translator.go

```diff
@@ -218,20 +230,23 @@ func (ds *sqlDatastoreTranslator) translatePrepared(input data.Node) (q string,
 			values = append(values, v.String())
 			operands.AppendToTop(getPreparePlaceholderForPlatform(ds.platform, len(values)))
 		default:
-			logging.LogForComponent("sqlDatastoreTranslator").Warnf("Unexpected input: %T -> %+v", v, v)
+			return errors.Errorf("Unexpected input: %T -> %+v", v, v)
```

Comment:
I would be careful with this change, because I think we have a some "Unexpected input" warnings.
So I would leave it as a warning and not transform it into an error until it is one.

Reply:
Do we have the possibility to output the package name here as well? Maybe we can help the developer to find the package that is using the unsupported function.

Reply:
I thought that a problem inside my translation engine should result in an error http response and not continue translating? So the client/developer is indicated that there is something wrong.
Package name could be added by editing error string or wrapping error.

Reply:
After a short discussion with @aobermair we found out that I mixed-up this default statement with the default statement in l. 79 of the internal.pkg.translate.default-processor.go which is a warning which cannot be transformed into an error.

So we are good to go here :)